actionpack/test/dispatch/ssl_test.rb

```diff
@@ -21,7 +21,7 @@ def assert_not_redirected(url, headers: {}, redirect: {})
   end
 
   def assert_redirected(redirect: {}, from: "http://a/b?c=d", to: from.sub("http", "https"))
-    redirect = { status: 301, body: [] }.merge(redirect)
```

Comment:
This status here is incorrect since it overrides the default HTTP status code used in the middleware.